frontend/public/components/nav/section.tsx

```diff
@@ -192,7 +188,7 @@ export const NavSection = connect(navSectionStateToProps)(
           const childItems = sortExtensionItems(this.getNavItemExtensions(perspective, title, id));
 
           childItems.forEach((item) => {
-            const pluginChild = this.mapChild(createLink(item as NavItem));
+            const pluginChild = this.mapChild(createLink(item as LoadedExtension<NavItem>));
```

Comment:
if you change `sortExtensionItems` to
```js
export const sortExtensionItems = <E extends NavItem>(
  extensionItems: LoadedExtension<E>[],
): LoadedExtension<E>[] => {
...
};
```

you will not have to cast to `LoadedExtension<NavItem>` anymore